src/pages/packer/plugins/index.tsx

```diff
@@ -64,11 +66,20 @@ async function getStaticProps(ctx) {
 		// Prepare nav data for client (such as adding `fullPath`)
 		const { preparedItems: navData } = await prepareNavDataForClient({
 			basePaths: ['packer', 'plugins'],
-			nodes: rawNavData,
+			/**
+			 * TODO: fix up types related to MenuItem[] & EnrichedMenuItem[]
+			 * Task: https://app.asana.com/0/1202097197789424/1202405210286689/f
+			 */
+			nodes: rawNavData as $TSFixMe as NavNode[],
 		})
 
-		// Replace our original navData with our prepared navData
-		staticProps.props.layoutProps.sidebarNavDataLevels[2].menuItems = navData
+		/**
+		 * Replace our original navData with our prepared navData
+		 * TODO: fix up types related to MenuItem[] & EnrichedMenuItem[]
```

Comment:
🙋🏻 Did you mean `EnrichedNavItem`? Could you also share your thoughts behind what needs to change about it? It was added as part of the efforts to start removing `MenuItem`, so I think I see it staying as-is and extended if needed.